manifests/broker/service.pp

```diff
@@ -43,29 +43,55 @@
     }
     $environment = deep_merge($env_defaults, $env)
 
-    if $::service_provider == 'systemd' {
-      include ::systemd
+    case $::service_provider {
+      'systemd': {
+        include ::systemd
 
-      file { "/etc/systemd/system/${service_name}.service":
-        ensure  => file,
-        mode    => '0644',
-        content => template('kafka/unit.erb'),
-      }
+        file { "/etc/systemd/system/${service_name}.service":
+          ensure  => file,
+          mode    => '0644',
+          content => template('kafka/unit.erb'),
+        }
 
-      file { "/etc/init.d/${service_name}":
-        ensure => absent,
-      }
+        file { "/etc/init.d/${service_name}":
+          ensure => absent,
+        }
 
-      File["/etc/systemd/system/${service_name}.service"]
-      ~> Exec['systemctl-daemon-reload']
-      -> Service[$service_name]
+        File["/etc/systemd/system/${service_name}.service"]
+        ~> Exec['systemctl-daemon-reload']
+        -> Service[$service_name]
 
-    } else {
-      file { "/etc/init.d/${service_name}":
-        ensure  => file,
-        mode    => '0755',
-        content => template('kafka/init.erb'),
-        before  => Service[$service_name],
+      }
+      'upstart': {
+        file { "/etc/init.d/${service_name}":
+	          ensure => absent,
+	        } ~>
```

Comment:
Is the spacing here out of whack/too deep?

Reply:
Yep